sources/packages/clamAV/Dockerfile.dev

```diff
@@ -0,0 +1,6 @@
+FROM clamav/clamav:1.2
+RUN freshclam
```

Comment:
Is it going to run `freshclam` in the background doing regular database upgrades (daemon mode)?

Reply:
Yes this is just to start the freshclam update when the container starts